### ISSUE TITLE
Test data in language model

### ISSUE THREAD

**Issue by djstrong** [zalandoresearch/flair#178]
Is somewhere used test data?
https://github.com/zalandoresearch/flair/blob/d7c0f1733b398d59b3bffb62f5a422343c7e420e/flair/trainers/language_model_trainer.py#L35

**Comment by stefan-it** [comment]
No, after the training there was no evaluation on test set in the recent language model I trained :(

**Comment by alanakbik** [comment]
Oh yes this is a mistake - we need to add a final evaluation on test data after all epochs or on exit from the loop to the `LanguageModelTrainer`. Thanks for spotting this @djstrong and @stefan-it

@stefan-it how are the language models coming? :)

**Comment by stefan-it** [comment]
Two weeks ago training worked, but now (with new GPU + own compiled `pytorch` version) it is currently not working, see #180
